modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -561,3 +561,66 @@ UPDATE QRTZ_TRIGGERS SET TRIGGER_STATE = 'WAITING' WHERE TRIGGER_STATE = 'PAUSED
 ----
 
 After this operation, the table QRTZ_PAUSED_TRIGGER_GRPS should be empty, and all the triggers in the QRTZ_TRIGGERS table should be in state _waiting_, and not _paused_.
+
+[#fk_troubleshoot]
+=== Some foreign keys are duplicated
+
+_Symptom_: After migrating to 7.11.0+, prior to 7.11.6, after re-installing/updating the BDM, some foreign key constants are effectively duplicated.
```

Comment:
Explain how to detect this duplication. An normal user will never see this duplication, and everything will work fine.